oauth2client/contrib/django_util/__init__.py

```diff
@@ -236,16 +313,47 @@ def __init__(self, settings_instance):
             _get_oauth2_client_id_and_secret(settings_instance)
 
         if ('django.contrib.sessions.middleware.SessionMiddleware'
-                not in settings_instance.MIDDLEWARE_CLASSES):
-            raise exceptions.ImproperlyConfigured(
-                "The Google OAuth2 Helper requires session middleware to "
-                "be installed. Edit your MIDDLEWARE_CLASSES setting"
-                " to include 'django.contrib.sessions.middleware."
-                "SessionMiddleware'.")
+           not in settings_instance.MIDDLEWARE_CLASSES):
+                raise exceptions.ImproperlyConfigured(
```

Comment:
done, I think? tricky to get right since not on next line wants a visual indent of 3 spaces and raise should be 4 spaces from the original if. At least intellij flake8 is telling me this is correct.